packages/integration_test/lib/integration_test.dart

```diff
@@ -235,6 +235,7 @@ https://flutter.dev/docs/testing/integration-tests#testing-on-firebase-test-lab
   Future<void> enableTimeline({
     List<String> streams = const <String>['all'],
     @visibleForTesting vm.VmService? vmService,
+    @visibleForTesting HttpClient? httpClient,
```

Comment:
fwiw, this is a public API and saying @visibleForTesting doesn't really prevent people from using the parameters. We should consider these to just be part of the public API, document them, and so on.